source/boards/data-and-archives.rst

```diff
@@ -0,0 +1,142 @@
+Import your data
+================
+
+You can import data from other tools to use with Boards. [Contribute code](https://www.focalboard.com/contribute/getting-started/) to expand this.
+
+Import from Asana
+-----------------
+
+This node app converts an Asana JSON archive into a ``.focalboard`` archive.
```

Comment:
Question: Is .focalboard the new or old archive format?

Reply:
That's a great question. @sbishel - should this be ``.focalboard`` or ``.boardarchive``?

Reply:
@sbishel, ping on this. :)